lib/cucumber-rest-bdd/resource.rb

```diff
@@ -18,6 +20,16 @@
     steps %Q{When I send a GET request to "#{url}"}
 end
 
+When(/^I request (?:an?|the) (.+?)(?: with (?:key|id))? "([^"]*)" with:$/) do |resource, token, params|
+    resource_name = get_resource(resource)
+    url = get_url("#{resource_name}/#{token}")
+    unless params.rows_hash.empty?
+        query = params.rows_hash.map{|key, value| %/#{get_parameter(key)}=#{resolve(value)}/}.join("&")
```

Comment:
Would `raw` instead of `rows_hash` make more sense here? e.g. if i wanted to test duplicate params: `http://url?query=1&query=2`